libs/langchain-community/src/vectorstores/azure_aisearch.ts

```diff
@@ -368,7 +369,8 @@ export class AzureAISearchVectorStore extends VectorStore {
     k = 4,
     filter: this["FilterType"] | undefined = undefined
   ): Promise<[Document, number][]> {
-    const searchType = this.options.type ?? AzureAISearchQueryType.Similarity;
+    const searchType =
+      this.options.type ?? AzureAISearchQueryType.SimilarityHybrid;
```

Comment:
What's the reason for this change? Technically this is "breaking" to users in the sense that the functionality will be different.

Reply:
Hybrid search is the default retreival mode for other vector stores that support it. We'd like to align to that pattern as hybrid proves better retrevial quality vs pure vector search.

Reply:
We had a discussion with the PMs regarding which search mode would bring the best relevance in most scenarios as a default value, and concluded that hybrid search should be the best for most use cases.
This is technically breaking, but people relying on defaults should get slightly improved relevance without any other impact on their code.